library/src/main/java/com/heartlandpaymentsystems/library/terminals/c2x/C2XDevice.java

```diff
@@ -216,6 +216,7 @@ public void setConnectionConfig(ConnectionConfig connectionConfig) {
         this.connectionConfig = connectionConfig;
 
         LibraryConfigHelper.setDebugMode(connectionConfig.getEnvironment().equals(Environment.TEST));
+        LibraryConfigHelper.setAllowDupes(connectionConfig.getAllowDupes());
```

Comment:
Are we limited to setting the AllowDupes flag at the config layer only? With our other SDKs, this is a transaction-level flag that can be set per request as needed.

Reply:
I'll update so it can be set per transaction.